src/components/shared/MarkerTooltipContents.js

```diff
@@ -34,6 +47,50 @@ function getMarkerDetails(data: MarkerPayload): React$Element<*> | null {
           </div>
         );
       }
+      case 'GCMinor': {
+        if (data.nursery && data.nursery.reason) {
```

Comment:
I really like Flow coverage over Firefox's data because we can explicitly state our assumptions. If something can be null, definitely express that in the Flow types. If we are reasonably confident it won't ever be null, we should explicitly state that in the type, but if we ever get a runtime error we can adjust our types, and update our assumptions. Flow will be strict about making sure were update our code wherever we've used those types.